tb/cocotb/soc/test_pll_apb_regs.py

```
@@ -153,47 +158,82 @@ async def test_pll_apb_regs_status_locked(dut):
+    await ClockCycles(dut.clk, 2)
+
+    data2, ok3 = await bfm.read(CTRL_OFFSET)
+    assert ok3, "CONTROL readback (pass 2) returned pslverr=1 (SLVERR)"
+
+    assert (data2 & 0x1) == 1, (
+        f"GH #89 VIOLATION: pll_enable (CONTROL[0]) was cleared by a firmware write — "
+        f"WMASK[0]=0 must preserve reset value 1; "
+        f"got CONTROL=0x{data2:08x}"
+    )
+    assert int(dut.pll_enable_o.value) == 1, (
+        f"GH #89 VIOLATION: pll_enable_o dropped to 0 after write-zero — "
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)